Makefile

```diff
@@ -11,7 +11,8 @@ deps:
 build: $(wildcard **/*.go)
 	@echo "Building CLI..."
 	@mkdir -p build
-	go build -o build/linux_amd64
+	GOOS=linux GOARCH=amd64 go build -o build/linux_amd64 .
+	GOOS=darwin GOARCH=amd64 go build -o build/darwin_amd64 .
```

Comment:
I decided it will be beneficial to have OS X version as part of the build as well :).